README.md

```diff
@@ -0,0 +1,97 @@
+# appu
+
+Appu is a toolkit for podcast edition and publishing.
```

Comment:
**A**utomatic **P**odcat **PU**blisher, aka appu, is a toolkit for podcast edition and publishing.